web/modules/custom/collection/src/Form/JoinCollectionForm.php

```diff
@@ -50,26 +50,48 @@ public function buildForm(array $form, FormStateInterface $form_state, AccountPr
     // confirmation form, disguised as a form submit button. The confirmation
     // form should open in a modal dialog for JavaScript-enabled browsers.
     if (Og::isMember($collection, $user)) {
-      $form['leave'] = [
-        '#type' => 'link',
-        '#title' => $this->t('Leave this collection'),
-        '#url' => Url::fromRoute('collection.leave_confirm_form', [
-          'rdf_entity' => $collection->id(),
-        ]),
-        '#attributes' => [
-          'class' => [
-            'use-ajax',
-            'button',
-            'button--small',
-            'button--blue-light',
-            'mdl-button',
-            'mdl-js-button',
-            'mdl-button--raised',
-            'mdl-js-ripple-effect',
-            'mdl-button--accent',
+      $form['leave_wrapper'] = [
+        '#type' => 'dropbutton',
```

Comment:
Hi @kalinchernev ,
there is already dropdown button in drupal, it is something different than mdl dropdown (there is an example on add custom page form). 
On the designs there is no "You're a member" text, only "Leave collection" button (or "Join collection"). 
I thought that the additional text "You're a member" should be also styled like button, so it would be consistent with other buttons which are in this place (right part of the header).
I used dropdown button, but because "You're a member" is not a real button, it is not clickable (only arrow, because it dropdown buttons only arrow triggers dropdown). So this button is disabled and because of that it has different color. If needed I could remove this disable state (so entire button would trigger dropdown) or change the color for disable state.

I can also change this dropdown and use mdl dropdown. But I'm not sure how should look the "You're a member" text. Label, white area, button?